src/route/routesSSRData.js

```diff
@@ -228,21 +234,31 @@ export const libraryPage = async (store, match, show_console = false) => {
   }
 
   show_console && console.log('serverRender: libraryPage source was found', sourceID);
+  const sourceIndexAction  = sourceIndex(sourceID);
+  const id                 = getSourceIndexId(sourceIndexAction);
+  const sourceIndexPromise = new Promise(resolve => {
```

Comment:
Not sure what you did here.
Looks like you manually build the data without the data moving through redux, is that correct?.

Reply:
Yep, exactly, manual fetch-and-use + push to redux too